force_bdss/core/data_value.py

```diff
@@ -0,0 +1,34 @@
+from traits.api import HasStrictTraits, Any, String, Enum
+
+
+class DataValue(HasStrictTraits):
```

Comment:
I think it can be useful for the GUI, when displaying the results in the table we want to be able to distinguish KPIs to other parameters. But I agree that it's better if the author of the plugin don't have to deal with this.